index.html

```diff
@@ -225,6 +225,29 @@
 				return uniqueFeatures;
 			}
 
+			function updateURL(district=null) {
+				newURL = window.location.pathname;
+				newTitle = 'Raise Your Hand Texas programs'
```

Comment:
Note that `newTitle` will overwrite the current window title, after having some relevant context appended to it.  So this is where to edit if you want the title changed.